presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -108,15 +110,17 @@ public PlanOptimizers(
             SqlParser sqlParser,
             FeaturesConfig featuresConfig,
             MBeanExporter exporter,
+            CostComparator costComparator,
             StatsCalculator statsCalculator,
             CostCalculator costCalculator,
-            CostCalculator estimatedExchangesCostCalculator)
+            @EstimatedExchanges CostCalculator estimatedExchangesCostCalculator)
```

Comment:
Shouldn't both `CostCalculator` be explicitly annotated? Shouldn't both be named `somethingSomethingCostCalculator`?